### ISSUE TITLE
Missing a JSON Wire endpoint leads to failure

### ISSUE THREAD

**Issue by und3fined-v01d** [presidenten/wdio-video-reporter#45]
**Describe the bug**
We have a pattern match check for jsonWireMsg endpoint in wdio-video-reporter `index.js` file at https://github.com/presidenten/wdio-video-reporter/blob/2e113f17a45f7ef345f4cf14f2bbbfc904e1f604/src/index.js#L86. 

However if an action has no endpoint, the video reporter reports an error.

**Log**

>23:58:24 [0-0] 2020-06-15T18:28:24.306Z ERROR @wdio/sync: TypeError: Cannot read property 'match' of undefined
23:58:24     at Video.onAfterCommand (/workspace/src/extensions/AbuseFilter/node_modules/wdio-video-reporter/dist/wdio-video-reporter.js:486:42)
23:58:24     at reporters.forEach.reporter (/workspace/src/extensions/AbuseFilter/node_modules/@wdio/runner/build/reporter.js:38:49)
23:58:24     at Array.forEach (<anonymous>)
23:58:24     at BaseReporter.emit (/workspace/src/extensions/AbuseFilter/node_modules/@wdio/runner/build/reporter.js:38:20)
23:58:24     at EventEmitter.browser.on.result (/workspace/src/extensions/AbuseFilter/node_modules/@wdio/runner/build/index.js:196:50)
23:58:24     at Browser.WebDriver.prototype.(anonymous function) [as emit] (/workspace/src/extensions/AbuseFilter/node_modules/@wdio/utils/build/monad.js:133:33)
23:58:24     at saveScreenshot (/workspace/src/extensions/AbuseFilter/node_modules/wdio-mediawiki/index.js:23:11)
23:58:24     at Object.afterTest (/workspace/src/extensions/AbuseFilter/tests/selenium/wdio.conf.js:98:21)

**Expected behavior**
A video should be recorded without any errors.

**Suggested fix**
The reporter should not emit this error.

Currently, we do something like this:
```js
const command = jsonWireMsg.endpoint.match(/[^\/]+$/);
```

Where we should be doing a check like this:
```js
const command = jsonWireMsg.endpoint && jsonWireMsg.endpoint.match(/[^\/]+$/);
```

**Steps to reproduce**
1. Clone Mediawiki/core repository using:
`git clone https://github.com/wikimedia/mediawiki mediawiki`
2. cd mediawiki
3. Run `git clone https://github.com/wikimedia/mediawiki-extensions-AbuseFilter extensions/AbuseFilter`
4. Append the line `wfLoadExtension( 'AbuseFilter' ); ` in `LocalSettings.php`
5. Run `git clone https://github.com/wikimedia/Vector skins/Vector`
6. Set up mediawiki as specified in `DEVELOPERS.md`
7. Check if AbuseFilter is visible on navigation to https://localhost:8080/wiki/Special:Version
8. cd extensions/AbuseFilter
9. npm i && npm run selenium-test

**Environment (please complete the following information):**
 - **wdio-video-reporter version:** 3
 - **WebdriverIO version:** 6
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** sync/async
 - **Node.js version:** 12.x
 - **NPM version:** 6.x
 - **Browser name and version:** Chrome 83
 - **Platform name and version:** OSX Cataline

**Desktop (please complete the following information):**
 - OS: OS
 - Browser Chrome
 - Version 83

**Comment by und3fined-v01d** [comment]
@presidenten I am not sure how to appropriately title this, but this should be an easy PR.

**Comment by presidenten** [comment]
Thank you for a very nicely described issue!
Just curious though, what kind of actions do not correspond to an endpoint?

**Comment by presidenten** [comment]
(Merged your pr in v3.1.0)

**Comment by und3fined-v01d** [comment]
> Thank you for a very nicely described issue!
> Just curious though, what kind of actions do not correspond to an endpoint?

I guess those instances where we perform a script injection using `browser.execute` don't get registered as JSON wire endpoint. I was planning on doing a short POC on confirming the same. Hopefully with v3.1 we should be able to confirm this behaviour.
